apps/server/src/api/controllers/inventory-movement/register-inbound-inventory-movement-controller.ts

```diff
@@ -0,0 +1,17 @@
+import { batchRepository, inventorymovementRepository } from "@/database";
+import { HTTP_STATUS_CODE } from "@stocker/core/constants";
+import { BatchDto, InventoryMovementDto } from "@stocker/core/dtos";
+import { IHttp } from "@stocker/core/interfaces";
+import { RegisterInboundInventoryMovementUseCase } from "@stocker/core/use-cases";
+
+
+export class RegisterInboundInventoryMovementController {
+  async handle(http: IHttp) {
+    const batchDto = http.getBody<BatchDto>()
+    const inventoryMovementDto = http.getBody<InventoryMovementDto>()
```

Comment:
merge the `batchDto` and `InventoryMovementDto` in a single body object type lke below:
```
type Body = {
  batch: BatchDto
  inventoryMovement: InventoryMovementDto
}
```